projects/ngx-advanced-img/src/lib/classes/bitmap.ts

```diff
@@ -212,6 +213,30 @@ export class NgxAdvancedImgBitmap {
     this.debug = false;
   }
 
+  /**
+   * Standard function to convert a HEIC image to a different format for further processing.
+   * 
+   * @param heic The HEIC file to convert.
+   * @param format The format to convert the HEIC file to.
+   */
+  public static convertHEIC(blob: Blob, format: 'image/jpeg' | 'image/png'): Promise<Blob> {
```

Comment:
Since we aren't using externally, this should probably be private. No action required.